packages/linux-firmware/slice-firmware/package.mk

```diff
@@ -33,13 +33,13 @@ PKG_AUTORECONF="no"
 
 make_target() {
   if [ "$DEVICE" = "Slice3" ]; then
```

Comment:
No need to check $DEVICE here, as we only build slice-firmware for suitable devices.

Reply:
Slice and Slice3 have different firmware .. hence the check

Reply:
Ah yes...